static/js/components/email/EmailCompositionDialog_test.js

```diff
@@ -64,6 +65,35 @@ describe('EmailCompositionDialog', () => {
     assert.equal(document.querySelector('h3').textContent, 'Test Email Dialog');
   });
 
+  it('renders radio button for email type config', () => {
```

Comment:
could we name this test more explicitly? something like `renders a radio button for setting whether an email is automatic or not` or similar.